src/containers/ContainerButtonPanel.js

```diff
@@ -20,8 +20,21 @@ class ContainerButtonPanel extends React.PureComponent {
     enabledKeys: PropTypes.bool,
   };
 
+  componentDidUpdate(prevProps) {
+    const { enabledKeys } = this.props;
+
+    if (prevProps.enabledKeys !== enabledKeys) {
```

Comment:
Please avoid using nested ifs and else statements. This lifecycle can be rewritten as:
```
componentDidUpdate(prevProps) {
  const { enabledKeys } = this.props;
  const { enabledKeys: prevEnabledKeys } = prevProps;
  const eventHandler = enabledKeys ?
    document.removeEventListener :
    document.addEventListener;

  if (prevEnabledKeys !== enabledKeys) {
    eventHandler('keydown', this.handleKey);
  }
}
```